env.yml.example

```diff
@@ -0,0 +1,45 @@
+# neo4j authentication
+neo4j:
+  username: neo4j
+  password: change-me!
+  uri: bolt://localhost:7687
+  http_port: 7474
+  bolt_port: 7687
+
+# backend LLM properties
+mistral_key: my-api-key
+
+# OpenAI LLM properties
+openai_key: my-openai-api-key
+
+# frontend and backend URLs
+frontend_url: http://localhost:8650
+backend_url: http://localhost:8250
+ws_url: ws://localhost:8250/ws
+
+# Azure
+azure:
+  storage_connection_string: "my-connection-string"
+  storage_container_name: my-container-name
+  initial_data_filename: test-data.json
+
+# llm config
```

Comment:
Might have been nicer to merge the Agents, LLMs and Models e.g.
```
agents:
  - agent: "answer"
    llm: "mistral"
    model: "mistral-large-latest"
  - agent: intent"
    llm: "mistral"
    model: "mistral-large-latest"
```
etc. etc.

This would keep everything for an agent together so you wouldn't need to scroll up and down and will help to not miss something out when adding new agents.

Reply:
@philbaxter - I have made the changes according to your suggestion.